tabs/tab_corr.py

```
@@ -10,20 +10,28 @@
 Updated: Uses dataset selector pattern like tab_diag.py
 """
 
-from shiny import ui, reactive, render, req, module
-from utils.plotly_html_renderer import plotly_figure_to_html
-import pandas as pd
+import html as _html
+import os
+import re
+import tempfile
+from typing import Any, Dict, List, Optional, Union, cast
```

Ruff (0.14.13)

`typing.Dict` is deprecated, use `dict` instead

(UP035)